src/engine/media/widget.php

```diff
@@ -341,7 +341,14 @@ function ($url, $code) {
     'embed.youtube' => array(
         '(?:www\\.youtube\\.com/watch\\?|youtu\\.be/([-0-9A-Za-z_]{1,64}))',
         function ($logger, $match, $query) {
-            $start = isset($query['t']) && is_numeric($query['t']) ? (int)$query['t'] : 0;
+            $start = 0;          
+            if (isset($query['t'])) {
+                $t_param = $query['t'];
+                if (strpos($t_param,"s")==strlen($t_param)-1){
+                    $t_param = substr($t_param,0,strlen($t_param)-1); 
+                }
```

Comment:
[OPT] As an alternative you can leverage `substr`'s ability to deal with negative indices here:
```suggestion
                if (substr($t_param, -1) === 's') {
                    $t_param = substr($t_param, 0, -1); 
                }
```
Mind the triple "===" to perform a strict comparison, even though there should be no risk of nasty automatic cast here 🙂